src/pages/settings/ServerSettings.tsx

```diff
@@ -111,6 +122,26 @@ export default observer(() => {
             category="server_pages"
             switchPage={switchPage}
             defaultPage="overview"
+            custom={
+                owner ? (
+                    <>
+                        <LineDivider />
+                        <ButtonItem
+                            onClick={() =>
+                                openScreen({
+                                    id: "special_prompt",
+                                    type: "delete_server",
+                                    target: server,
+                                })
+                            }
+                            className={styles.logOut}
```

Comment:
I'm guessing this was meant to be `styles.deleteServer`? I will just edit and merge.